src/server/models/cacheable_queries/opt-out.js

```diff
@@ -184,7 +185,7 @@ const optOutCache = {
       const org =
         organization || organizationCache.load(campaign.organization_id);
       await processServiceManagers("onOptOut", org, {
-        contact: newContact,
```

Comment:
thank you for these fixes! @crayolakat and @Arique1104  -- even if the rest of this PR isn't merged, this file's changes are bug fixes

Reply:
The changes in this file have been merged into Spoke version 12